nasa-gibs/worldview
# WV-3040: Update VIIRS Day/Night Band Nighttime Lights version

## Description

Fixes #WV-3040 .

- Updated VIIRS_SNPP_DayNightBand_At_Sensor_Radiance and VIIRS_SNPP_DayNightBand_AtSensor_M15 to version 2 in layer description. 
- Changed revisit period of HLS Landsat layers to 8 day from 16 day.

## How To Test

1. 'git checkout wv-2040-update-dnb-version'
2. 'npm run build && npm start'
3. Open with [these URL parameters](http://localhost:3000/?v=-136.54691126931823,36.74332917391904,-46.337720937089706,77.89482699558887&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,VIIRS_SNPP_DayNightBand_AtSensor_M15,VIIRS_SNPP_DayNightBand_At_Sensor_Radiance,VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor(hidden)&lg=true&t=2024-02-11-T15%3A32%3A26Z)
4. Verify expected result - make sure the link in the DOI points to version 2


## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
